autobisect/evaluator/browser.py

```diff
@@ -89,80 +90,64 @@ def evaluate_testcase(self, build_path):
         result = Bisector.BUILD_FAILED
         with self.prefs() as prefs_file:
             if os.path.isfile(binary) and self.verify_build(binary, prefs=prefs_file):
-                for _ in range(self.repeat):
-                    log.info("> Launching build with testcase...")
-                    result = self.launch(binary, self.testcase, prefs_file)
-                    if result == Bisector.BUILD_CRASHED:
-                        break
+                log.info("> Launching build with testcase...")
+                result = self.launch(binary, self.testcase, prefs_file)
 
         return result
 
-    def launch(self, binary, testcase, prefs=None):
+    def launch(self, binary, test_path, prefs=None):
         """
         Launch firefox using the supplied binary and testcase
         :param binary: The path to the firefox binary
-        :param testcase: The path to the testcase
+        :param test_path: The path to the testcase
         :param prefs: The path to the prefs file
         :return: The return code or None
         """
-        ffp = FFPuppet(
-            use_gdb=self._use_gdb,
-            use_valgrind=self._use_valgrind,
-            use_xvfb=self._use_xvfb,
-        )
-        if self._asserts:
-            ffp.add_abort_token("###!!! ASSERTION:")
-
-        result = Bisector.BUILD_PASSED
+        replay = None
+        target = None
+        testcase = TestCase.load_path(test_path)
+        if self._env_vars:
```

Comment:
Add a comment here because usually this is unnecessary unless you plan to report the test case.

Reply:
Bah this is not pointing to the correct lines... I am referring to adding the environment vars to the test case.

Reply:
As discussed, this is actually correct.  The environment variables are added to the testcase which is then used by ReplayManager.